wayback/_client.py

```diff
@@ -492,7 +495,12 @@ def search(self, url, *, matchType=None, limit=None, offset=None,
                 if isinstance(value, str):
                     final_query[key] = value
                 elif isinstance(value, datetime):
-                    final_query[key] = value.strftime(URL_DATE_FORMAT)
+                    # Make sure we have either a naive datetime (assumed to
+                    # represent UTC) or convert the datetime to UTC.
+                    value_utc = value
+                    if value.tzinfo:
+                        value_utc = value.astimezone(tzutc())
```

Comment:
I *think* we can just use `value.astimezone(datetime.timezone.utc)` here, right? That way we don’t need the extra `dateutil` dependency. (Guess I should have done that in the original.)

Reply:
Verified interactively and pushed.